Valigator.Newtonsoft.Json/ValigatorConverter.cs

```diff
@@ -14,6 +14,16 @@ public class ValigatorConverter : JsonConverter
 	{
 		private ConcurrentDictionary<Type, IConverter> _converters = new ConcurrentDictionary<Type, IConverter>();
 
+		private readonly JsonSerializerSettings _jsonSerializerSettings;
+
+		private JsonSerializer _serializer;
+		public JsonSerializer Serializer => _serializer ?? (_serializer = JsonSerializer.Create(_jsonSerializerSettings));
```

Comment:
The serializer that will be used to (de)serializing. This is to avoid the throwing an exception in a location that cannot be handled.